templates/implnet_ops_SOURCEVAL.py

```diff
@@ -119,82 +141,46 @@ def strict_env(key: str):
 
 
 def _graphEndpoint():
-    url = f"{GLEANER_GRAPH_URL}/namespace/{GLEANER_GRAPH_NAMESPACE}/sparql"
-    return url
-
+    return f"{GLEANER_GRAPH_URL}/namespace/{GLEANER_GRAPH_NAMESPACE}/sparql"
 
 def _graphSummaryEndpoint():
-    url = f"{GLEANER_GRAPH_URL}/namespace/{GLEANER_GRAPH_NAMESPACE}_summary/sparql"
-    return url
-
+    return f"{GLEANER_GRAPH_URL}/namespace/{GLEANER_GRAPH_NAMESPACE}_summary/sparql"
 
 def _pythonMinioAddress(url, port=None) -> str:
     """Construct a string for connecting to a minio S3 server"""
     if not url:
         raise RuntimeError("Tried to construct minio address with an empty URL")
-    get_dagster_logger().info(f"{url=}{port=}")
-
-    if url.endswith(".amazonaws.com"):
-        PYTHON_MINIO_URL = "s3.amazonaws.com"
```

Comment:
Leaving it as client responsibility to give the right url. I dont think we want to special case check all these s3 urls